manager/forms.py

```diff
@@ -193,6 +195,61 @@ class Meta:
             'applies_to'
         )
 
+class ReportDetailForm(forms.Form):
+    email_select = forms.ModelMultipleChoiceField(label="Email(s)",
+        queryset=Email.objects.all(),
+        help_text='Select the emails which data will be aggregegated from.',
+        to_field_name='pk')
+
+    start_date = forms.DateField(label="Start Date",
+        help_text='The start date to pull data from.',
+        required=True,
+        initial=date.today() - timedelta(days=90))
```

Comment:
`start_date` does not have `strftime()` formatting applied to it, but `end_date` does--is this intentional?